docs/source/hackrf_one.rst

```diff
@@ -25,4 +25,16 @@ Features
     * pin headers for expansion
     * portable
     * open source
-    
+
+
+
+Minimum System Requirements for using HackRF
```

Comment:
I disagree because the minimum requirements talk about what the host computer needs to support both the HackRF hardware and the SDR software someone might choose to run. The information isn't specific to the HackRF hardware itself  which is what the "hardware" section refers to.

Reply:
The point I'm making is that the information in minimum system requirements is distinct and important enough to have its own page.
The minimum system requirements are currently nested on the HackRF One page under Hardware. If that information isn't specific to the HackRF hardware, then that isn't the right place either.